examples/with-universal-configuration/package.json

```diff
@@ -3,7 +3,9 @@
   "version": "1.0.0",
   "scripts": {
     "dev": "next",
-    "build": "next build",
+    "clean": "rimraf .\\node_modules\\.cache",
+    "build-dev": "npm run clean && next build --env=development",
```

Comment:
maybe we need to pass additional parameters like instance (we need to pass 2 params)